### DISCUSSION TITLE
No search result for new Articles + Returning Irrelevant Search Results

### DISCUSSION THREAD

**Discussion by AHTei** [microsoft-search/pnp-modern-search#4016] [Q&A]
**Version used**
Ex: 4.3.0.0

**Describe the bug**
No search result for new Articles + Returned Irrelevant Search Results

**ISSUE #1:**
Search does not return result for new items when search a field value (#3961) raised issue earlier is still present.
We are using PnP-Modern-Search V4 for Site Pages where we have added several metadata fields including a text field for "Article ID".
We have migrated article contents from different environment into the SharePoint Site Pages and are able to search them successfully using Article ID values. However, when a new page (article) is created, and Article ID gets a new unique value along its name then searching for this article works fine using its Name but not using its Article ID value. We already verified that the Article ID value is present in the related article data load contents.
Furthermore, we raised a Case with Microsoft SharePoint Support Team, and they confirmed the normal SharePoint Search is able to find the article using its Article ID value. If we use the Mapped Property name as a label before the value for Article ID in the search box it also finds the article in the PnP Search. Related image is attached.

**ISSUE#2:**
When entering a search term like 4442 as Article ID of a Site Page in the search box, then it results to displaying 7 articles where 6 of them are irrelevant with no relation to Article ID 4442 and do not have the search term highlighted in those 6 irrelevant items.
 
**Expected behavior**
1- Terms in the newly created Articles need to be searchable using custom metadata like Article ID.
2- No additional irrelevant resuts should be returned.

**Screenshots**
Proof of working OOB Search for the ISSUE#1.

**Desktop (please complete the following information):**
 - Browser (Microsoft Edge and Google Chrome)
![Working-OOB-Search](https://github.com/user-attachments/assets/fd8fbdf9-1d71-4f8f-99a1-f9df1d5c8e77)

**Comment by kasperbolarsen** [comment]
Which version are you using? We can only support the most resent version.

However, since you can search and find the item in issue 1 using out of the box MS search, you should have a closer look at properties of the managed property, in the schema in the SharePoint Admin center. Can you share the settings?
Issue2: That is up to you to ensure the search query is specific enough to only return relevant results. If you only want results where the text from the search box matches the ArticleID when the query should be something like  ArticleIDOWSText:{searchTerm}

**Comment by AHTei** [comment]
We have updated to the latest Release 4.13.1 version and still have the same reported issues as well as getting duplicated results in the search summaries.

Please find attached requested screen captures.

![0-Default Managed Property based on Site Column Created by SharePoint](https://github.com/user-attachments/assets/d4db4428-7d14-4f36-9d0c-f37c7a039dde)
![0-Managed Property Created](https://github.com/user-attachments/assets/2671aae4-b747-4a3d-ba16-cf0896dad6ac)
![0-Site Collection Search Setting](https://github.com/user-attachments/assets/25159c1e-3f0f-411d-8127-51865d72a4f1)
![1-No Result for Newly Created Article](https://github.com/user-attachments/assets/f652b6e9-96e3-4ca4-8c03-e13ac7dc46fc)
![2-New Article Expected Good Result](https://github.com/user-attachments/assets/bf75c39c-915e-48bb-acb3-4f7630c2e470)
![3-Search Irrelevent Results](https://github.com/user-attachments/assets/8234e140-ac5b-4f67-a20c-8a6292a720b3)
![4-Search Expected Result](https://github.com/user-attachments/assets/71783a4d-f806-4efd-bb49-bde0810c69c8)

**Comment by AHTei** [comment]
Hi @kasperbolarsen,
Please provide your guidance how to address these issues with priority as they are two of the defects blocking our project delivery to the end of the line and we are already behind the schedule.
I appreciate your kind understanding. Thanks. --Amir

**Comment by wobba** [comment]
Curious if this post solves your scenario: https://www.techmikael.com/2020/03/side-effect-of-mapping-crawled.html

**Reply by kasperbolarsen** [reply]
My thoughts exactly
